packages/block-library/src/image/save.js

```diff
@@ -49,20 +49,26 @@ export default function save( { attributes } ) {
 		[ `wp-image-${ id }` ]: !! id,
 	} );
 
+	const imgStyle = {
+		...borderProps.style,
+		aspectRatio,
+		objectFit: scale,
+	};
+	if ( width ) {
+		imgStyle.width = width;
+	}
+
+	if ( height && ! isNaN( height ) ) {
```

Comment:
Why might height be `NaN`? We should try to prevent that from happening rather than checking `isNaN()`.

Reply:
Height can be 'auto', so maybe we should just check for that.